platform_plugin_aspects/views.py

```diff
@@ -108,7 +125,8 @@ def get(self, request, *args, **kwargs):
         """
         course = self.get_object()
 
-        dashboards = settings.ASPECTS_INSTRUCTOR_DASHBOARDS
+        dashboards = settings.ASPECTS_INSTRUCTOR_DASHBOARDS[:]
```

Comment:
Is the `[:]` needed here? If it's just for the copy, can you use an explicit `settings.ASPECTS_INSTRUCTOR_DASHBOARDS.copy()` for readability?